README.md

```diff
@@ -1,82 +1,82 @@
-=Wagn: how pioneers roll
+#Wagn: how pioneers roll
 
-==Basics
+[![Build Status](https://semaphoreapp.com/api/v1/projects/0d61c1f9-5ef0-4c5c-89c6-7664e247e4be/294670/badge.png)](https://semaphoreapp.com/ethan/wagn)
```

Comment:
> It should always be green on master, and if it isn't a core dev better fix it soon.

Mostly true.  But master is the new develop, and the gem is the new master.  If master gets broken, we need to fix it in a hurry.   But whereas master needs to be green, say 98% of the time, the current gem version needs to be working 99.99% of the time.  (And we don't want to risk losing the users who want to install during the 2% red time.)